### PR TITLE
adding badge

### PR DESCRIPTION
**Pull request by sanjay-xdr**
added badge icon & avatar
badge-cl

### CODE REVIEW

File: Component/Badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+.avatar-lg {
+  height: 10rem;
+}
+.avatar-md {
+  height: 6rem;
+}
+
+.avatar-sm {
+  height: 3rem;
```
Comment (Mohasin-Haque) [inline Component/Badge/badge.css:31]: Run prettier on this in order to indent the code properly;

File: Component/Badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+.box {
+  height: 14rem;
+  width: 120rem;
+  justify-content: space-around;
+  align-items: center;
+  padding: 2rem;
+}
+
+figure {
```
Comment (devdeadviz) [inline Component/Badge/badge.css:11]: avoid using direct tag selector to apply styles

File: Component/Badge/badge.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="../../Utility CSS/utility.css" />
+    <link rel="stylesheet" href="./badge.css" />
```
Comment (devdeadviz) [inline Component/Badge/badge.html:12]: Try to import only a single stylesheet and put others' stylesheets in that same stylesheet that you have linked to your HTML.

File: Component/Badge/badge.html [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+  <body>
+    <header>
+      <nav class="container-flex flex-sb flex-wr">
+        <div class="header__nav">
+          <ul class="container-flex">
+            <li>
+              <a
+                href="../../index.html"
+                class="font-l anchor-style nav__icon--position nav__item--hover"
+                >F#
```
Comment (devdeadviz) [inline Component/Badge/badge.html:26]: What is this F#?

File: Component/Badge/badge.html [hunk 5]
```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
```
Comment (Mohasin-Haque) [inline Component/Badge/badge.html:8]: Add meta descriptions to make it more personalise like.
<meta name="author" content="your name" />
    <meta name="description"
        content="description of library." />

File: Component/Badge/badge.html [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+            <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28231%2C229%2C229%2C1%29&t=one-light&wt=none&l=text%2Fx-csrc&width=617.25&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=143%25&si=false&es=2x&wm=false&code=%253Cfigure%253E%250A%2520%253Cimg%2520class%253D%2522round-img%2520avatar-lg%2522%2520src%253D%2522......%2522%2520alt%253D%2522%2522%2520%252F%253E%250A%2520%253Cdiv%2520class%253D%2522badge%2520badge-online%2522%253E%253C%252Fdiv%253E%250A%253C%252Ffigure%253E%250A%250A%253Cfigure%253E%250A%2520%253Cimg%2520class%253D%2522round-img%2520avatar-lg%2522%2520src%253D%2522......%2522%2520alt%253D%2522%2522%2520%252F%253E%250A%2520%253Cdiv%2520class%253D%2522badge%2520badge-offline%2522%253E%253C%252Fdiv%253E%250A%253C%252Ffigure%253E%250A%250A%253Cfigure%253E%250A%2520%253Cimg%2520class%253D%2522round-img%2520avatar-lg%2522%2520src%253D%2522......%2522%2520alt%253D%2522%2522%2520%252F%253E%250A%2520%253Cdiv%2520class%253D%2522badge%2520badge-dnd%2522%253E%253C%252Fdiv%253E%250A%253C%252Ffigure%253E"
+            style="width: 100%; height: 337px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (devdeadviz) [inline Component/Badge/badge.html:124]: Avoid using inline styles.

File: Component/Badge/badge.html [hunk 7]
```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
```
Comment (Mohasin-Haque) [inline Component/Badge/badge.html:9]: You can add it to main styling file so that you need not to write it everywhere

**Review by devdeadviz — COMMENTED** [review]
Suggested some changes which you can have a look at.

**Review by Mohasin-Haque — CHANGES_REQUESTED** [review]
Your code looks so clean and tidy but needs to take care of the commented problems.. You are good to go ahead once you make changes.

File: Component/Badge/badge.css (hunk 1 above)
Comment (sanjay-xdr) [reply Component/Badge/badge.css:31]: sure

File: Component/Badge/badge.html (hunk 7 above)
Comment (sanjay-xdr) [reply Component/Badge/badge.html:9]: yes will work on this as i have done same with every component so at the end will fix it

File: Component/Badge/badge.html (hunk 3 above)
Comment (sanjay-xdr) [reply Component/Badge/badge.html:12]: yes

File: Component/Badge/badge.css (hunk 2 above)
Comment (sanjay-xdr) [reply Component/Badge/badge.css:11]: ok have to fix this for all the components then will fix it at the end

**Pull request merged**